File: app/src/main/java/com/adsaff/mywebrtc/MainActivity.java
package com.adsaff.mywebrtc;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.EditText;
import android.widget.ImageView;
import android.widget.Toast;

import org.json.JSONException;
import org.json.JSONObject;

import java.net.URISyntaxException;

import io.socket.client.IO;
import io.socket.client.Socket;
import io.socket.emitter.Emitter;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ImageView imageView = findViewById(R.id.connect_button);
        EditText editText = findViewById(R.id.room_edittext);
        imageView.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                if(editText.getText().toString().isEmpty()){
                   Toast.makeText(getApplicationContext(),"Please enter a room Id",Toast.LENGTH_LONG).show();
                }else {
                    startActivity(new Intent(getApplicationContext(),WebSocketActivityKubma.class).putExtra("room_id",editText.getText().toString()));
                }
            }
        });
    }
}

File: app/src/main/java/com/adsaff/mywebrtc/MainActivity.kt
package com.adsaff.mywebrtc

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.view.View
import android.view.inputmethod.InputMethodManager
import android.widget.EditText
import android.widget.ImageView
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        callToWebRTCMethod()
    }

    private fun hideKeyboardFrom(context: Context, view: View) {
        val imm = context.getSystemService(INPUT_METHOD_SERVICE) as InputMethodManager
        imm.hideSoftInputFromWindow(view.windowToken, 0)
    }

    private fun callToWebRTCMethod() {
        val imageView = findViewById<ImageView>(R.id.connect_button)
        val editText = findViewById<EditText>(R.id.et_room_id)
        imageView.setOnClickListener { view ->
            if (editText.text.toString().isEmpty()) {
                Toast.makeText(applicationContext, R.string.please_enter_a_room_id, Toast.LENGTH_LONG).show()
            } else {
                hideKeyboardFrom(this@MainActivity, view)
                startActivity(Intent(applicationContext, WebRTCActivity::class.java).putExtra("room_id", editText.text.toString()))
            }
        }
    }
}